templates/index/genesis.html

```diff
@@ -1,15 +1,19 @@
 {{ define "genesis" }}
   <div class="row pt-3">
-    <div class="col-lg-5 my-2">{{ template "depositProgress" . }}</div>
+    <div class="col-lg-5 my-2">{{ template "depositProgress" .Data }}</div>
     <div class="col-lg-2 my-2"></div>
-    <div class="col-lg-5 my-2">{{ template "genesisCountdown" . }}</div>
+    <div class="col-lg-5 my-2">{{ template "genesisCountdown" .Data }}</div>
   </div>
   <div style="position:relative" class="card mt-3 index-stats">
     <div class="card-header pt-3">
       <div class="row">
         {{ template "networkStats" }}
       </div>
     </div>
+    <div class="mt-4">
+      <div class="w-100 my-2" id="slotsViz"></div>
+    </div>
+    {{ template "SlotViz" . }}
```

Comment:
> code line above not relevant for this

The slot viz is not shown if `.Data.SlotVizData` is nil, which happens if the config is not explicitly configured to show the slot viz. This shouldn't be necessary for the genesis index page, it should show the slot viz regardless (as it did before). You could change the `index.go` handler so that `.Data.SlotVizData` is also filled when the genesis period is active.